hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconUtils.java

```diff
@@ -245,6 +246,16 @@ public InputStream makeHttpCall(CloseableHttpClient httpClient, String url)
     }
   }
 
+  public InputStream makeHttpCall(URLConnectionFactory connectionFactory,
```

Comment:
Can we remove the other makeHttpCall that accepts CloseableHttpClient if it is no longer used?